inference-engine/src/vpu/graph_transformer/src/frontend/detect_network_batch.cpp

```diff
@@ -82,94 +63,66 @@ void FrontEnd::detectNetworkBatch(
 
         auto ieShapes = ieData->getTensorDesc().getDims();
         env.log->trace("Input [%s] : %v", p.first, ieShapes);
+        // assume only 4D and 5D inputs have batch
+        if (ieShapes.size() == 4 || ieShapes.size() == 5) {
```

Comment:
Should we remove `DETECT_NETWORK_BATCH` config from tests when testing NC layouts?

Reply:
I don't think so.
In the case with NC layout, we just call `return` on line 89.